simulator/src/test/java/com/hazelcast/simulator/probes/impl/HdrProbeTest.java

```diff
@@ -26,16 +30,19 @@ public void testConstructor_noThroughputProbe() {
         assertFalse(tmpProbe.isPartOfTotalThroughput());
     }
 
+    // this test is fragile and can easily fail if there is some kind of pausing going on
```

Comment:
This can't be tested reliably. Imagine you get a gc pause, then the recorded time will be a lot larger than expected.